src/course-home/outline-tab/alerts/certificate-available-alert/CertificateAvailableAlert.jsx

```diff
@@ -1,50 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { getConfig } from '@edx/frontend-platform';
-import { FormattedMessage, FormattedRelative } from '@edx/frontend-platform/i18n';
-import { Hyperlink } from '@edx/paragon';
+import { FormattedDate, FormattedMessage } from '@edx/frontend-platform/i18n';
 
 import { Alert, ALERT_TYPES } from '../../../../generic/user-messages';
 
 function CertificateAvailableAlert({ payload }) {
   const {
     certDate,
-    username,
-    userTimezone,
+    courseEndDate,
   } = payload;
 
-  const timezoneFormatArgs = userTimezone ? { timeZone: userTimezone } : {};
+  const certificateAvailableDate = <FormattedDate value={certDate} day="numeric" month="long" year="numeric" />;
+  const courseEndDateFormatted = <FormattedDate value={courseEndDate} day="numeric" month="long" year="numeric" />;
 
   return (
-    <Alert type={ALERT_TYPES.INFO}>
+    <Alert type={ALERT_TYPES.SUCCESS}>
       <strong>
         <FormattedMessage
           id="learning.outline.alert.cert.title"
-          defaultMessage="We are working on generating course certificates."
+          defaultMessage="Your grade and certificate will be ready soon!"
         />
       </strong>
       <br />
       <FormattedMessage
         id="learning.outline.alert.cert.when"
-        defaultMessage="If you have earned a certificate, you will be able to access it {timeRemaining}. You will also be able to view your certificates on your {profileLink}."
+        defaultMessage="This course ended on {courseEndDateFormatted} and final grades and certificates are
+        scheduled to be available after {certificateAvailableDate}."
```

Comment:
nit: this might read better as two sentences
```suggestion
        defaultMessage="This course ended on {courseEndDateFormatted}. Final grades and certificates are
        scheduled to be available after {certificateAvailableDate}."
```

Reply:
Any suggestions on language should be brought to the slack group please @kdmccormick. This language is from the UX team.